level.go

```diff
@@ -0,0 +1,24 @@
+package slogx
+
+import (
+	"log/slog"
+	"strings"
+)
+
+// ParseLevel convert levelName from flag or config file into slog.Level.
+func ParseLevel(levelName string) slog.Level {
+	switch strings.ToLower(strings.TrimSpace(levelName)) {
+	case "err", "error", "fatal", "crit", "critical", "alert", "emerg", "emergency":
+		return slog.LevelError
+	case "wrn", "warn", "warning":
+		return slog.LevelWarn
+	case "inf", "info", "notice":
+		return slog.LevelInfo
+	case "dbg", "debug", "trace":
+		return slog.LevelDebug
+
+	default:
+		slog.Debug("failed", "levelName", levelName) // Will be changed to use custom DefaultLogger.
```

Comment:
Before log/slog there was a clear rule: libraries must try hard to avoid logging anything. In the future this rule may be relaxed if slog will become common and standard enough, but today I don't think we should break this rule and start logging here and there with ease in libraries.

To me it looks like there are two use cases for this feature:
- Take string from flag/config/etc. and use it to set some log level without too much hassle. If ParseLevel will return error or `ok bool` or panic it won't be possible to use it without hassle, so to make this work it looks like best we can do is to return LevelDebug.
- Check is value from flag/config/etc. is a valid log level name and then decide how to handle it - use any default level or exit with error message etc. If someone will need this feature we can add another one function e.g. `TryParseLevel(string) (slog.Level, error)`.

So, let's just return LevelDebug without any logging.